Gemfile

```diff
@@ -21,8 +21,14 @@ gem "sass-rails", ">= 6"
 
 gem "bootstrap-sass", "3.4.1"
 
+gem "bootstrap-will_paginate", "1.0.0"
```

Comment:
remove it, are u using this gem ?
if remove it, the application work ok? right?

Reply:
![image](https://user-images.githubusercontent.com/45727724/90613601-31d24380-e23c-11ea-99dd-e1528d3fcdf7.png)
this error come out when i delete bootstrap -will paginate and run bundle install and restart server. How do i fix it with kaminari?

Reply:
nevermind. i solved.